### ISSUE TITLE
[Proposal] How about translating Wardrobe in other languages ?

### ISSUE THREAD

**Issue by ghost** [ericlbarnes/wardrobe#52]
Hi! 
I was wondering if you'd want to translate it in other languages. 
All the documentation and even the app itself front&backend of course.

I could do it in Romanian (native speaker) and in Italian (i live in Florence, Italy since aug 2006 and graduated here too).

What do you think?

**Comment by rtablada** [comment]
I think that would be very cool, I'm not sure what we would do for localization, I know Eric was looking at this a bit.

As far as documentation, maybe Eric can open up repos for docs-rom, docs-it.

I would be willing to shoot for Spanish. I am not a technical speaker but speak relatively fluently with my family daily.

**Comment by ghost** [comment]
IF he considers to do something about it count me in! It would be awesome.

**Comment by ericlbarnes** [comment]
I do like the idea and would like it to be external from the main repo. Let me do some thinking on it and devise a plan.

**Comment by rtablada** [comment]
I have seen Lang prefixed doc sets in other repo groups. And community Lang packs for internationalization
—

**Comment by ChristianGaertner** [comment]
Laravel does provide a pretty nifty Lang class (I' m sure most po you know that already) and I think rewriting the app to use the lang class should be pretty easy.
I' d like to help too btw :D (German).

**Comment by ericlbarnes** [comment]
I already converted the app to a Lang file. See Lang/en/wardrobe

Thanks,
Eric Barnes

**Comment by ericlbarnes** [comment]
I added a "locales" repo to the wardrobecms organization. This will be for translating the core. 
https://github.com/wardrobecms/locales

I will work on the docs repo over the next few days to get it working with sub directories for the different languages. Because I will not be able to audit different language files I will still only pull the english docs to the actual website. But will add information and links on finding the other docs. 

In the beginning I think the best way for this to work is just with pull requests against these repos.

**Comment by rtablada** [comment]
Sounds good, I'll start to get to work on Spanish.

Have a blessed day,
Ryan Tablada
ryan.tablada@gmail.com

—
Sent from Mailbox for iPhone

On Sun, Jul 21, 2013 at 1:54 PM, Eric Barnes <notifications@github.com="mailto:notifications@github.com">> wrote:
I added a "locales" repo to the wardrobecms organization. This will be for translating the core. 

I will work on the docs repo over the next few days to get it working with sub directories for the different languages. Because I will not be able to audit different language files I will still only pull the english docs to the actual website. But will add information and links on finding the other docs. 

In the beginning I think the best way for this to work is just with pull requests against these repos.

**Comment by ghost** [comment]
I'll start on Romanian and Italian then if that's ok with you

On Sunday, July 21, 2013, Ryan Tablada wrote:

> Sounds good, I'll start to get to work on Spanish.
> 
> Have a blessed day,
> Ryan Tablada
> ryan.tablada@gmail.com <javascript:_e({}, 'cvml',
> 'ryan.tablada@gmail.com');>
> 
> —
> Sent from Mailbox for iPhone
> 
> On Sun, Jul 21, 2013 at 1:54 PM, Eric Barnes <notifications@github.com<javascript:_e({}, 'cvml', 'notifications@github.com');>
> ="mailto:notifications@github.com <javascript:_e({}, 'cvml',
> 'notifications@github.com');>">> wrote:
> I added a "locales" repo to the wardrobecms organization. This will be for
> translating the core.
> 
> I will work on the docs repo over the next few days to get it working with
> sub directories for the different languages. Because I will not be able to
> audit different language files I will still only pull the english docs to
> the actual website. But will add information and links on finding the other
> docs.
> 
> In the beginning I think the best way for this to work is just with pull
> requests against these repos.
> 
> —
> Reply to this email directly or view it on GitHub.
> 
> —
> Reply to this email directly or view it on GitHubhttps://github.com/ericbarnes/wardrobe/issues/52#issuecomment-21314849
> .

## 

This email (including any attachment) is a personal message and may contain
confidential and/or privileged and/or proprietary information. If you have
received this email in error, please notify the sender immediately, do not
use or share it and destroy this email. Any unauthorised use, copying or
disclosure of the material in this email or of parts hereof (including
reliance thereon) is strictly forbidden.
I have taken precautions to minimize the risk of transmitting software
viruses but nevertheless advise you to carry out your own virus checks on
any attachment of this message. I accept no liability for loss or damage
caused by software viruses.

---

Il presente messaggio, inclusi gli eventuali allegati, ha natura personale
e potrebbe contenere informazioni confidenziali e/o riservate. Chiunque lo
ricevesse per errore, è pregato di avvisare tempestivamente il mittente e
di cancellarlo.
E’ strettamente vietata qualsiasi forma di utilizzo, riproduzione o
diffusione non autorizzata del contenuto di questo messaggio o di parte di
esso.
Pur essendo state assunte le dovute precauzioni per ridurre al minimo il
rischio di trasmissione di virus, si suggerisce di effettuare gli opportuni
controlli sui documenti allegati al presente messaggio. Non si assume
alcuna responsabilità per eventuali danni o perdite derivanti dalla
presenza di virus.

**Comment by ChristianGaertner** [comment]
Should we use the html encoded characters or can we use the UTF-8 way:

e.g.
    ö <=> `&ouml;`
    ü <=> `&uuml;`
    etc.

**Comment by ericlbarnes** [comment]
I am pretty sure you can use the first way. 

```
 'key' => 'öÀç'
```

**Comment by rtablada** [comment]
I am about halfway through the spanish translation so far. I'm realizing that there is a lot of technical things that I do not run into talking to my family and friends.

**Comment by Franarroutado** [comment]
I'm a spanish native speaker (Spain) and I would love to improve some translations if you don't mind.

**Comment by ChristianGaertner** [comment]
This is GitHub. Just send a pull request...

**Comment by ericlbarnes** [comment]
Yes just send a pull request.  All language files are in a secondary repo - https://github.com/wardrobecms/locales

**Comment by rtablada** [comment]
@Franarroutado thanks, I am a conversational speaker around the house so my grammar and technical knowledge in Spanish isn't too great.

I just wanted to get some rough translations that would be a start.

If you don't know how to do a pull request, send me the updates in an issue on my repo of the locales and I will submit a new pull request for Spanish.

**Comment by Franarroutado** [comment]
Thanks, I will make pull request with the changes.  @rtablada you really did a great job on the translation.

**Comment by mpmont** [comment]
I'll translate it to Portuguese in the near future. =)
